apps/omg_api/lib/state/transaction.ex

```diff
@@ -199,6 +199,15 @@ defmodule OMG.API.State.Transaction do
   defp parse_address(<<_::160>> = address_bytes), do: {:ok, address_bytes}
   defp parse_address(_), do: {:error, :malformed_address}
 
+  def decode(signed_tx_bytes) do
```

Comment:
I presume the input here isn't `signed_tx_bytes` but rather `raw_tx_bytes`.

Also please try to make naming and flow consistent with the version for `Transaction.Signed`. One imporant thing is `from_rlp` can't be included in the `try-rescue` block, see `Transaction.Signed.decode` and friends